app/presenters/business_presenter.rb

```diff
@@ -0,0 +1,9 @@
+module BusinessPresenter
+  def display_phone
+    self.raw_data['display_phone'] || self.raw_data['display_phone'] = "No Number Provided"
```

Comment:
Line is too long. [91/80]<br>Redundant `self` detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.